### PR TITLE
Couchbase Service Guide 0.2.0-5.5.0

### PR DESCRIPTION
**Pull request by ryadav88**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/couchbase/0.2.0-5.5.0/limitations/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Virtual networks
+
+When the service is deployed on a virtual network, the service may not be switched to host networking without a full re-installation. The same is true for attempting to switch from host to virtual networking.
+
+## Task Environment Variables
+
+Each service task has some number of environment variables, which are used to configure the task. These environment variables are set by the service scheduler. While it is possible to use these environment variables in ad hoc scripts (e.g., via `dcos task exec`), the name of a given environment variable may change between versions of a service and should not be considered a public API of the service.
+
+## Couchbase Specific
```
Comment (ryadav88) [inline pages/services/couchbase/0.2.0-5.5.0/limitations/index.md]: this section is missing

File: pages/services/couchbase/0.2.0-5.5.0/supported-versions/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Packages are versioned with an a.b.c-x.y.z format, where a.b.c is the version of the DC/OS integration and x.y.z indicates the version of Couchbase. For example, 0.1.1-2.3.2 indicates version 0.1.1 of the DC/OS integration and version 2.3.2 of Couchbase.
+
+## Version Policy
+
+The DC/OS Couchbase Service is engineered and tested to work with a specific release of Couchbase, AlertManager and PushGateway.
+
+## Contacting Technical Support
+
+### Support Email of DCOS Couchbase Package
```
Comment (ryadav88) [inline pages/services/couchbase/0.2.0-5.5.0/supported-versions/index.md]: lets also add a section for Couchbase support after we confirm with them

File: pages/services/couchbase/0.2.0-5.5.0/release-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+title: Release Notes
+menuWeight: 130
+excerpt: Release Notes for version 0.2.0-5.5.0
+featureMaturity:
+enterprise: false
+---
+
+Release notes for Couchbase.
+
+# Version 0.2.0-5.5.0
```
Comment (ryadav88) [inline pages/services/couchbase/0.2.0-5.5.0/release-notes/index.md]: Add release notes

File: pages/services/couchbase/0.2.0-5.5.0/index.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+DC/OS Couchbase Service is an autonomous service that makes it easy to deploy and operate Couchbase Server and the Couchbase Sync Gateway on Mesosphere [DC/OS](https://mesosphere.com/product/). For more information on Couchbase, see the [Couchbase documentation](https://developer.couchbase.com/documentation/server/current/introduction/intro.html).
+
+## Benefits
+DC/OS Couchbase offers the following benefits :
+1. Based on the battle proven DC/OS framework/operator sdk that underpins all DC/OS certified frameworks
+2. Supporting Couchbase Server
+3. Supporting Couchbase Sync Gateway
+4. Automated cluster deployment, production deployments and compact development deployments
```
[the remark this answers is not in the document]
Comment (realmbgl) [reply pages/services/couchbase/0.2.0-5.5.0/index.md]: done

File: pages/services/couchbase/0.2.0-5.5.0/quick-start-guide/index.md [hunk 5]
```diff
@@ -0,0 +1,220 @@
+---
+layout: layout.pug
+navigationTitle: Quick Start
+excerpt: How to use Couchbase with DC/OS
+title: Quick Start
+menuWeight: 15
+---
+
+This section gives a quick end 2 end tour on how to configure and use Couchbase Server and Couchbase Sync Gateway with DC/OS.
```
[the remark this answers is not in the document]
Comment (realmbgl) [reply pages/services/couchbase/0.2.0-5.5.0/quick-start-guide/index.md]: done

File: pages/services/couchbase/0.2.0-5.5.0/quick-start-guide/index.md [hunk 6]
```diff
@@ -0,0 +1,220 @@
...
+Next create a file named todo.json, with the following content. This is the pouchdb getting started app that accesses the sync gateway.
+
+
+```json
+{
+  "id": "todo",
+  "container": {
+    "type": "MESOS",
+    "docker": {
+      "image": "realmbgl/pdbtd",
```
[the remark this answers is not in the document]
Comment (realmbgl) [reply pages/services/couchbase/0.2.0-5.5.0/quick-start-guide/index.md]: done

**Pull request merged**